src/presenter/main-presenter.js

```diff
@@ -13,6 +13,18 @@ import { Sorts } from '../settings';
 import { SortSettings } from '../settings';
 import dayjs from 'dayjs';
 
+const getFilterSettings = (filter) => {
+  const nullPredicate = () => true;
+  const currentDate = dayjs();
```

Comment:
```suggestion
const nullPredicate = () => true;
const getFilterSettings = (filter) => {
  const currentDate = dayjs();
```
пусть компилятор видит, что `nullPredicate` не меняется при каждом вызове `getFilterSettings`